### PR TITLE
Initial public version (0.6) of the Solution Architecture.

### PR DESCRIPTION
**Pull request by ijansch**
Initial version of the solution architecture (work in progress!) document.

### CODE REVIEW

File: architecture/Solution Architecture.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+# COVID-19 Notification App - Solution Architecture
+
+## Baseline for the "Proof of Concept"
+
+**Version:** 0.6 (28/5/2020)
+
+**Status: ** Draft
```
Comment (bwbroersma) [inline architecture/Solution Architecture.md:7]: Should be `**Status:**` (no space), see [this markdown online](https://github.com/ijansch/nl-covid19-notification-app-coordination/blob/initial_version/architecture/Solution%20Architecture.md) to see the current rendering.

File: architecture/Solution Architecture.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+This document describes the functional and technical architecture of this PoC. Note that, as part of the Dutch Covid19 response, there are also several other workstreams happening in parallel (such as for example investigating a follow up, post infection, support app). 
+
+The requirements document received from the Dutch health authority GGD are leading in the decisions in this solution architecture. The original requirements document can be found at [https://www.rijksoverheid.nl/binaries/rijksoverheid/documenten/publicaties/2020/05/19/programma-van-eisen/20200519+Programma+van+Eisen+def.pdf](https://www.rijksoverheid.nl/binaries/rijksoverheid/documenten/publicaties/2020/05/19/programma-van-eisen/20200519+Programma+van+Eisen+def.pdf)
+
+**This document only details the proof of concept for exposure notification.**
+
+# Table of contents
+
+[[TOC]]
```
Comment (bwbroersma) [inline architecture/Solution Architecture.md:21]: This won't work out of the box on Github, see [your current markdown online](https://github.com/ijansch/nl-covid19-notification-app-coordination/blob/initial_version/architecture/Solution%20Architecture.md#table-of-contents).

File: architecture/Solution Architecture.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+# Baseline Approach
+
+This chapter describes the core flow that we are following, which is partially dictated by the choice of protocol.
+
+## Recording encounters 
+
+As described in the GAEN document set - the users’ mobile phones are able to record ephemeral IDs of other phones they meet. These are generated from a daily seed key (Temporary Exposure Key, TEK). Details about this can be found in the GAEN documentation, but it boils down to the following:
+
+![image alt text](images/image_0.png)
```
Comment (bwbroersma) [inline architecture/Solution Architecture.md:97]: Please use proper alt text. Would be great to use PlantText instead, see [this repo](https://github.com/openstate/govhub/blob/470d2b02423316f822c5109a5531f05d4e8ad23a/README.md#system-overview) for example, which is rendered from a [plain text file](https://github.com/openstate/govhub/blob/470d2b02423316f822c5109a5531f05d4e8ad23a/docs/system_overview.puml).

Comment (ijansch) [reply architecture/Solution Architecture.md:97]: Thanks! I will need to fix this later, I'm using a generator for the MD file and this is hard to fix manually at the moment. I've created an issue for it here: https://github.com/minvws/nl-covid19-notification-app-coordination/issues/9

File: architecture/Solution Architecture.md (hunk 1 above)
Comment (ijansch) [reply architecture/Solution Architecture.md]: Thanks, fixed

File: architecture/Solution Architecture.md (hunk 2 above)
Comment (ijansch) [reply architecture/Solution Architecture.md]: Thanks, fixed!

**Comment by ijansch** [comment]
Force pushed to ensure my commits were gpg signed.

**Pull request merged**